tests/internal/cluster/cluster.go

```diff
@@ -171,3 +179,209 @@ func ExecCmdWithStdout(
 
 	return outputMap, nil
 }
+
+// new commands
+
+// ExecLocalCommand runs the provided command with the provided args locally, cancelling execution if it exceeds
+// timeout.
+func ExecLocalCommand(timeout time.Duration, command string, args ...string) (string, error) {
+	ctx, cancel := context.WithTimeout(context.TODO(), timeout)
+	defer cancel()
+
+	glog.V(params.LogLevel).Infof("Locally executing command '%s' with args '%v'", command, args)
```

Comment:
Please move this log message to the beginning of the function.

Reply:
fixed